### DISCUSSION TITLE
Local storage instead of S3 Bucket (or equivalent)

### DISCUSSION THREAD

**Discussion by feloeht** [OpnForm/OpnForm#163] [Q&A]
Hi everyone, I'm trying to add a logo and cover image to my form, but I can't do it because the application seems to be waiting for an S3 bucket as storage.

I've tried changing the FILESYSTEM parameters (DRIVER and DISK) in the environment file for local and public, as specified in the laravel documentation, but that doesn't change anything, the logs display an S3 error every time I upload.

Do I have to create an S3 bucket (AWS or MINio) in order to store a simple logo?

Thanks!

**Comment by JhumanJ** [comment]
Hey, we just added support to local filestorage in https://github.com/JhumanJ/OpnForm/pull/165
